features/guests_added.feature

```diff
@@ -11,15 +11,46 @@ Feature: See if you are attending an event
       | Other event       | Look, it changes colors     | 22-Apr-3019 | 422 Treeside Way, Berkeley, CA 94704   | stubhub.com/liquid    | 0        |
     And I am on the events page
 
-  Scenario: I register for an event:
+
+
+  Scenario: I register for an event with no demographics:
+    When I follow "rsvp_1"
+    Then I should see "Gender"
+    And I fill out the form with the following attributes:
+      | firstname    | Test           |
+      | lastname     | Guest          |
+      | email        | test@guest.com |
+    Then I should see "You have successfully registered!"
+    And I should see "Test Guest"
+
+  Scenario: Delete a guest
```

Comment:
scenario does not match test so idk what this does

Reply:
Read the entire scenario.  There is just an extra space in between so it's easier to read

Reply:
you need to create a new scenario or add this to step def cause the first half does not test if a guest is deleted.

Reply:
This is not testing deleting a guest, this is testing registering with no demographics.  Look below for the delete scenario.

Reply:
Hence my point: the scenario is for deleting a guest. review how to write/organize cucumber tests.